java/src/main/java/ai/onnxruntime/OnnxRuntime.java

```diff
@@ -22,6 +22,8 @@
 
   // The initial release of the ORT API.
   private static final int ORT_API_VERSION_1 = 1;
+  // Post 1.0 builds of the ORT API.
+  private static final int ORT_API_VERSION_2 = 2;
```

Comment:
Why 2? We're using version 3 now.

Reply:
Not yet. Hari is changing it to 3. It should be 3.

Reply:
We started adding version 3 APIs before that. https://github.com/microsoft/onnxruntime/blob/06fc9506fdf83082f1ef226324fab31806dce1ce/onnxruntime/core/session/onnxruntime_c_api.cc#L1514

Reply:
Yeah but the header file still says 2 (https://github.com/microsoft/onnxruntime/blob/06fc9506fdf83082f1ef226324fab31806dce1ce/include/onnxruntime/core/session/onnxruntime_c_api.h#L10). When the changes that add the metadata accessors get merged in it'll be bumped to 3, and I'll bump it in the Java PR that exposes the metadata.